main/kcal-culator.py

```diff
@@ -0,0 +1,44 @@
+#[x] create a program that take a file location
```

Comment:
What do you think of using an `_` for the file name instead of a `-`? Using a `-` will make it difficult to import the file.

Reply:
Alright! :)